src/test/resources/simulators/adb.ts

```diff
@@ -35,19 +36,21 @@ interface IRunningApplicationState {
     isInDebugMode: boolean;
 }
 
-/* Simulation of DeviceHelper/ADB */
-export class DeviceHelper implements deviceHelper.IDeviceHelper {
+/* Simulation of adb/ADB */
+export class Adb extends adb.AdbEnhancements {
```

Comment:
Can you rename this to `AdbSimulation` or something? When I was reading the code further up I was confused that there was this "new Adb" being given a FileSystem object, when the (non-test) Adb class expects different arguments. Took me a bit to realise that they were different things with the same name.